bin/hdfcoinc/pycbc_coinc_findtrigs

```diff
@@ -126,8 +126,12 @@ class ReadByTemplate(object):
 
         if self.bank is not None:
             self.param = {}
-            for col in self.bank:
-                self.param[col] = self.bank[col][self.template_num]
+            if 'compressed_waveforms' in self.bank :
```

Comment:
@ahnitz So, do you want it to be something like :
```
if self.bank.attrs['parameters'] :
```
instead of 
```
if 'compressed_waveforms' in self.bank :
``` ?